spec.bs

```diff
@@ -663,6 +664,11 @@ dictionary AuctionAdConfig {
 };
 </xmp>
 
+Note: To help with ease of adoption, until TBD we will support the attribute

```

Comment:
just to double check that it is actually "TBD"

Reply:
We do not currently have a timeline for its removal. Though after k-anonymity enforcement hits 100% it will stop being useful.